cmd/kube-apiserver/app/server.go

```diff
@@ -125,6 +127,8 @@ func (s *APIServer) AddFlags(fs *pflag.FlagSet) {
 		"Read only port will be opened on this address, and it is assumed that port "+
 		"443 at this address will be proxied/redirected to '-address':'-port'. If "+
 		"blank, the address in the first listed interface will be used.")
+	fs.BoolVar(&s.ListenAllInterfaces, "listen_all_interfaces", s.ListenAllInterfaces, ""+
```

Comment:
Naively this overlaps completely with --address - why doesn't that work?

Perhaps we should rename flags and fields until we can look at this logic and know what it does?